packages/grid/_modules_/grid/hooks/features/columns/useColumns.ts

```diff
@@ -16,14 +16,47 @@ import { useApiMethod } from '../../root/useApiMethod';
 import { Logger, useLogger } from '../../utils/useLogger';
 import { useGridState } from '../core/useGridState';
 
+function mapColumns(
```

Comment:
Should this be selector?

Reply:
I would keep it as it is. This function is only used in the `hydrateColumns` one and initially, that functionality was part of the `hydrateColumns` but I separated it from it because the `hydrateColumns` was becoming a bit too bloated.